### PR TITLE
Rollbacks v1

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: notifications/notifications.yml [hunk 1]
```diff
@@ -25,6 +29,8 @@ Metadata:
     ParameterLabels:
       InfrastructureStorageStackName:
         default: Storage stack name
+      InfrastructureConfigStagingKey:
+        default: Production S3 object key
```
Comment (cavis) [inline notifications/notifications.yml]: Production?

File: notifications/notifications.yml [hunk 2]
```diff
@@ -44,6 +50,12 @@ Parameters:
     Default: infrastructure-storage
     Description: The name of a previously launched infrastructure storage stack
     Type: String
+  InfrastructureConfigStagingKey:
+    Default: template-config-staging.zip
+    Description: >
+      The S3 object key for the production template configuration file archive
```
Comment (cavis) [inline notifications/notifications.yml]: (again) production?

File: notifications/notifications.yml [hunk 3]
```diff
@@ -255,6 +322,31 @@ Resources:
       Dimensions:
         - Name: FunctionName
           Value: !Ref IkeCallbackHandlerLambdaFunction
+  IkeSlashCommandsHandlerErrorAlarm:
```
Comment (cavis) [inline notifications/notifications.yml]: Strange that this would ever be an issue.  But better than silently failing, i suppose.

**Comment by farski** [comment]
Fixed those descriptions @cavis

**Pull request merged**